### ISSUE TITLE
False positive tslint-plugin-prettier, tslint-react, tslint-sonarts

### ISSUE THREAD

**Issue by VincentLanglet** [depcheck/depcheck#314]
### Bug Description
Unused dependencies : 
```
* tslint-plugin-prettier
* tslint-react
* tslint-sonarts
```

Missing dependencies : 
```
* tslint-config-tslint:all
* tslint-config-tslint-react
* tslint-config-tslint-sonarts
```

### Code snippets
Where the dependency is used:

tslint.json
```
{
  "extends": ["tslint:all", "tslint-react", "tslint-sonarts", "tslint-config-prettier"],
  "rulesDirectory": ["tslint-plugin-prettier"],
  ...
}
```

Where the dependency is listed in package.json:
```
    "tslint": "^5.12.0",
    "tslint-config-prettier": "^1.18.0",
    "tslint-plugin-prettier": "^2.0.1",
    "tslint-react": "^3.6.0",
    "tslint-sonarts": "^1.8.0",
```

### Versions
 - `node -v`: v11.6.0
 - `npm -v`: 6.5.0-next.0
 - `depcheck --version`: 0.7.1

**Comment by VincentLanglet** [comment]
This still should be fixed

**Comment by rumpl** [comment]
Could you check with the latest depcheck please, I just added a testcase and it seems to work.

Thanks

**Comment by VincentLanglet** [comment]
I got unused with
```
* tslint-plugin-prettier
```

But i got unused too for
```
* prettier
```

This is maybe related.

I do have `.prettierrc` in my project

**Comment by rumpl** [comment]
And `tslint-plugin-prettier` is used in `rulesDirectory`?

**Comment by VincentLanglet** [comment]
`cat tslint.json` returns
```
{
  "extends": ["tslint:all", "tslint-react", "tslint-sonarts", "tslint-config-prettier"],
  "rulesDirectory": ["tslint-plugin-prettier"],
  "jsRules": true,
  "rules": {
    // From tslint
    "completed-docs": false,
    "file-name-casing": [true, { "^index.tsx|.ts$": "camel-case", ".tsx": "pascal-case" }],
    "interface-name": [true, "never-prefix"],
    "match-default-export-name": false,
    "no-default-import": false,
    "no-default-export": false,
    "no-import-side-effect": false,
    "no-magic-numbers": false,
    "no-null-keyword": false,
    "no-null-undefined-union": false,
    "no-submodule-imports": false,
    "no-use-before-declare": false,
    "no-unsafe-any": false,
    "no-void-expression": [true, "ignore-arrow-function-shorthand"],
    "strict-boolean-expressions": false,
    "typedef": [
      true,
      "call-signature",
      "arrow-call-signature",
      "parameter",
      "arrow-parameter",
      "property-declaration",
      "member-variable-declaration"
    ],
    "variable-name": [true, "allow-pascal-case"],

    // From plugin-prettier
    "prettier": true
  }
}
```

**Comment by rumpl** [comment]
Would it be possible to create a minimal repo that reproduces this bug please? That would be immensely helpful

**Comment by VincentLanglet** [comment]
https://github.com/VincentLanglet/testDepcheck

**Comment by rumpl** [comment]
Awesome, thanks!

**Comment by rumpl** [comment]
It's soooo much easier to debug with a real repo...

**Comment by rumpl** [comment]
Fixed, thanks!
